### PR TITLE
Initial changes for consistency in formatting and tightening syntax

### PR DESCRIPTION
**Pull request by christinerose**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
-Dune - A composable build system
+Dune - A Composable Build System
 ================================
 
 Dune is a build system designed for OCaml/Reason projects only. It
 focuses on providing the user with a consistent experience and takes
-care of most of the low-level details of OCaml compilation. All you
-have to do is provide a description of your project and dune will
+care of most low-level details of an OCaml compilation. It's merely necessary
```
Comment (jeremiedimino) [inline README.md:6]: I'm not sure about the `an` here. It was meant to talk about the process of compiling OCaml programs in general.

**Review by jeremiedimino — APPROVED** [review]
The new formulations look better to me!

Just left a comment about `an OCaml compilation`, which I'm not sure about

File: README.md (hunk 1 above)
Comment (christinerose) [reply README.md:6]: Perhaps "...care of most low-level details of OCaml compilations." ...make compilation plural? If that's still not accurate, it's fine to leave it as it is.

Comment (jeremiedimino) [reply README.md:6]: Yes, making it plural seems more natural to me.

**Comment by christinerose** [comment]
> The new formulations look better to me!
> 
> Just left a comment about `an OCaml compilation`, which I'm not sure about

Thanks! I'm glad they're looking better to you.

**Comment by christinerose** [comment]
I'm not sure why there are errors. I ran them again, but it looks like they're timing out (??)
I'll be pushing some more initial edits to other docs soon as well.

**Comment by jeremiedimino** [comment]
The problem seems unrelated, you can ignore it.

BTW, once a PR is approved, it is best to merge it and do new changes in a new PR. Otherwise it is difficult for reviewers to differentiate the old changes from the new ones. If you are still working on a PR and want to do more changes, you can open a "draft PR".

**Comment by christinerose** [comment]
Excellent! Thank you! I'll do that.

**Pull request merged**